### PR TITLE
Use fs-extra instead of ncp

### PR DESCRIPTION
**Pull request by ahmedalsudani**

### CODE REVIEW

**Comment by ahmedalsudani** [comment]
@fjvallarino @tejohnso ready for review

File: package.json [hunk 1]
```diff
@@ -19,11 +19,11 @@
     "simple-mock": "^0.4.1"
   },
   "dependencies": {
+    "fs-extra": "git+https://github.com/Rise-Vision/node-fs-extra",
```
Comment (ahmedalsudani) [inline package.json]: Patched version that requires our patched `graceful-fs` which imports `original-fs` in Electron

**Comment by tejohnso** [comment]
:+1:

**Comment by fjvallarino** [comment]
:+1:

**Pull request merged**